### PR TITLE
Feature/recipe repo test

### LINKED ISSUE #4: Build 'RecipeRepository' tests
- [x] Should have a list of all recipes
- [x] Should have a list of current recipes
- [x] Should have a list of filter terms
- [x] Should have a list of filter tags
- [x] Should be able to filter by favorite recipes
- [x] Should be able to filter recipes by category (tag)
- [x] Should be able to filter recipes by ingredient
- [x] Should be able to filter recipes by name
- [x] Should be able to return saved recipes
- [x] Should have a way to clear the filters

### PR DESCRIPTION
**Pull request by philalewis**
What (if any) features are you implementing?
- Build out recipe repo test file

What (if anything) did you refactor?
- New data in data file

Were there any issues that arose?
- No issues. We talked through our file paths and methods

Is there anything that you need from your teammate?
- Review and merge, please

Any other comments, questions, or concerns?
- nope

### CODE REVIEW

File: src/data/recipe-test-data.js [hunk 1]
```diff
@@ -1,45 +1,80 @@
...
-            "unit": "sl"
-          }
-        },
-        {
-          "id": 3,
-          "quantity": {
-            "amount": 1,
-            "unit": "tsp"
-          }
+  "image": "https://grillchesse.jpg",
```
Comment (mr-ryan12) [inline src/data/recipe-test-data.js:3]: Nice job adding extra recipes to test for variations in values.

File: test/RecipeRepository-test.js [hunk 2]
```diff
@@ -1,8 +1,92 @@
...
+  it('should be able to filter by multiple search terms', () => {
+    recipeRepo.addFilter('sandwich');
+    recipeRepo.filterRecipes();
+    expect(recipeRepo.currentRecipeIds.length).to.equal(2);
+    recipeRepo.addFilter('dinner');
+    recipeRepo.filterRecipes();
+    expect(recipeRepo.currentRecipeIds.length).to.equal(1);
+  });
+
+  it('should be able to clear search terms', () => {
```
Comment (mr-ryan12) [inline test/RecipeRepository-test.js:79]: I'm excited to see how this functionality works out.

**Comment by mr-ryan12** [comment]
Looks good! Thank you for doing this. Let's see how it performs with the class file.

File: src/data/recipe-test-data.js [hunk 3]
```diff
@@ -1,45 +1,80 @@
...
+    {
+      "instruction": "Butter two pieces of bread.",
+      "number": 1
+    },
+    {
+      "instruction": "Grill our bread with cheese in between.",
+      "number": 2
+    }
+  ],
+  "name": "Grill Cheese",
```
Comment (oacortez) [inline src/data/recipe-test-data.js:37]: Thank you for adding this new data set for our recipe data test

**Pull request merged**